server/rest/src/main/java/org/infinispan/rest/resources/ContainerResource.java

```diff
@@ -161,7 +161,18 @@ public Invocations getInvocations() {
       NettyRestResponse.Builder responseBuilder = checkCacheManager(request);
       if (responseBuilder.getHttpStatus() == NOT_FOUND) return completedFuture(responseBuilder.build());
 
-      return asJsonResponseFuture(cacheManager.getCacheManagerInfo().toJson(), responseBuilder);
+      Boolean rebalancingEnabled = null;
+      try {
+      rebalancingEnabled = SecurityActions.getGlobalComponentRegistry(cacheManager).getLocalTopologyManager()
+            .isRebalancingEnabled();
+      } catch (Exception e) {
+
+      }
+      Json cacheManagerInfo = cacheManager.getCacheManagerInfo().toJson();
+      if (rebalancingEnabled != null) {
+         cacheManagerInfo.set("rebalancing_enabled", rebalancingEnabled);
+      }
+      return asJsonResponseFuture(cacheManagerInfo, responseBuilder);
```

Comment:
I think we should add the rebalancing status to  `CacheManagerInfo` directly. That way embedded users can access this information as well and the `ContainerResource` is simpler.

Reply:
changed to cacheManagerinfo